instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -0,0 +1,647 @@
+            description="System disk time",
+            unit="seconds",
+        )
+
+        # self.accumulator.register_valueobserver(
+        #     callback=self._get_system_filesystem_usage,
+        #     name="system.filesystem.usage",
+        #     description="System filesystem usage",
+        #     unit="bytes",
+        #     value_type=int,
+        # )
+
+        # self._meter.create_observable_gauge(
+        #     callback=self._get_system_filesystem_utilization,
+        #     name="system.filesystem.utilization",
+        #     description="System filesystem utilization",
+        #     unit="1",
+        #     value_type=float,
+        # )
+
+        self._meter.create_observable_counter(
+            callback=self._get_system_network_dropped_packets,
+            name="system.network.dropped_packets",
+            description="System network dropped_packets",
+            unit="packets",
+        )
+
+        self._meter.create_observable_counter(
+            callback=self._get_system_network_packets,
+            name="system.network.packets",
+            description="System network packets",
+            unit="packets",
+        )
+
+        self._meter.create_observable_counter(
+            callback=self._get_system_network_errors,
+            name="system.network.errors",
+            description="System network errors",
+            unit="errors",
+        )
+
+        self._meter.create_observable_counter(
+            callback=self._get_system_network_io,
+            name="system.network.io",
+            description="System network io",
+            unit="bytes",
+        )
+
+        self._meter.create_observable_up_down_counter(
+            callback=self._get_system_network_connections,
+            name="system.network.connections",
+            description="System network connections",
+            unit="connections",
+        )
+
+        self._meter.create_observable_counter(
+            callback=self._get_runtime_memory,
+            name=f"runtime.{self._python_implementation}.memory",
+            description=f"Runtime {self._python_implementation} memory",
+            unit="bytes",
+        )
+
+        self._meter.create_observable_counter(
+            callback=self._get_runtime_cpu_time,
+            name=f"runtime.{self._python_implementation}.cpu_time",
+            description=f"Runtime {self._python_implementation} CPU time",
+            unit="seconds",
+        )
+
+        self._meter.create_observable_counter(
+            callback=self._get_runtime_gc_count,
+            name=f"runtime.{self._python_implementation}.gc_count",
+            description=f"Runtime {self._python_implementation} GC count",
+            unit="bytes",
+        )
+
+    def _uninstrument(self, **__):
+        pass
+
+    def _get_system_cpu_time(self) -> Iterable[Measurement]:
+        """Observer callback for system CPU time
+
+        Args:
+            observer: the observer to update
```

Comment:
Nit: Do callbacks have to have `args` docstrings?

Reply:
Good catch, removed.